templates/taxi/car_detail.html

```diff
@@ -3,20 +3,45 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  
+   <div class="d-flex justify-content-between align-items-center">
+    <h1>Drivers</h1>
+    {% if user.is_authenticated %}
+      {% if user in car.drivers.all %}
+        <form action="{% url 'taxi:remove_driver' car.id user.id %}" method="post">
```

Comment:
The URL identification parameter `name` in the `url` function should use `-` to split words, not `_`. This is according to point 5 of the Code Style section in the checklist. For example, it should be 'taxi:remove-driver' instead of 'taxi:remove_driver'.